README.md

```diff
@@ -37,8 +37,15 @@ See [the manpages](https://shopify.github.io/ecfg) for more technical documentat
   is `*{json,yaml,yml,toml}`, this flag is optional, but when the type can't
   be inferred because of a different filename or when reading from stdin, it
   must be specified. (note that this recognizes `*.ejson` as json automatically)
-* All references to EJSON have changed to ECFG, of course, including
-  `/opt/ecfg/keys` and `ECFG_KEYDIR`.
+* All references to EJSON have changed to ECFG, including `ECFG_KEYDIR`.
+* Key lookup semantics are a bit more complex: In addition to
```

Comment:
Key lookup has been changed to allow a greater degree of control.
- `ECFG_PRIVATE_KEY` may be set as an environment variable. This can be used to preempt key selection logic during decrypt. If present, ecfg will blindly attempt to decrypt any encrypted values using this key, instead of trying to find the matching key in the keydir. This is  useful for deploying to Heroku.
- Unless `ECFG_KEYDIR` is set, `ecfg` will now be look in four directories (in order of decreasing precedence):
  * `$XDG_CONFIG_HOME/ecfg/keys` (if set)
  * `$HOME/.ecfg/keys`
  * `/etc/ecfg/keys`
  * `/opt/ejson/keys`
- If `ECFG_KEYDIR` is set, then `ecfg` will look in `$ECFG_KEYDIR/keys`